build/Program.cs

```diff
@@ -132,32 +147,39 @@ await Cli.Wrap(dotnetExePath).WithArguments($"pack -noLogo {Option("-c", configu
                     .Select(PathString.New)
                     .ToArray();
                 foreach (var nupkgPath in nupkgPaths) {
-                    await Cli.Wrap(dotnetExePath).WithArguments($"nuget push \"{nupkgPath}\" " +
-                            $"--force-english-output " +
-                            $"--timeout 60 " +
-                            $"--api-key \"{nugetOrgApiKey}\" " +
-                            $"--source \"{feed}\" " +
-                            $"--skip-duplicate")
+                    await Cli.Wrap(dotnetExePath).WithArguments(new[] {
+                            "nuget",
+                            "push",
+                            nupkgPath,
+                            "--force-english-output",
+                            "--timeout", "60",
+                            "--api-key", nugetOrgApiKey,
+                            "--source", feed,
+                            "--skip-duplicate"
+                        })
                         .ToConsole()
                         .ExecuteAsync(cancellationToken)
-                        .Task.ConfigureAwait(false);
+                        .ConfigureAwait(false);
                 }
             });
 
             Target("coverage", DependsOn("build"), async () => {
                 CreateDir(testOutputPath);
                 var cmd = await Cli.Wrap(dotnetExePath)
-                    .WithArguments($"test " +
-                        "--nologo " +
-                        "--no-restore " +
-                        "--blame " +
-                        $"--collect:\"XPlat Code Coverage\" --results-directory {testOutputPath} " +
-                        $"{Option("-c", configuration)} " +
-                        $"{Option("-f", framework)} " +
-                        "-- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=json,cobertura"
+                    .WithArguments(args => args
+                        .Add("test")
+                        .Add("--nologo")
+                        .Add("--no-restore")
+                        .Add("--blame")
+                        .Add("--collect:\"XPlat Code Coverage\"")
+                        .Add("--results-directory").Add(testOutputPath)
+                        .AddOption("-c", configuration)
+                        .AddOption("-f", framework)
+                        .Add("--")
```

Comment:
Was this on purpose? The dashes were separated by a space from the following arguments in the original code.

Reply:
I think it was - they were passed as a single argument :) I.e. you're using an array syntax @vchirikov probably wasn't aware of...

Reply:
I was just making sure it's not meant to be

```
--DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=json,cobertura
```

instead of:

```
-- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=json,cobertura
```